### DISCUSSION TITLE
I can't create a kanban view

### DISCUSSION THREAD

**Discussion by cxzxlq** [nocodb/nocodb#5910] [Q&A] [answered]
I downloaded nocodb today, but when creating a kanban view, it cannot be created
![image](https://github.com/nocodb/nocodb/assets/65665060/5b00eba7-c082-4250-8630-583f67bcc87a)
How do I choose a Grouping Field

**Comment by wingkwong** [comment, marked as answer]
Currently we only allow Single Select as Grouping Field, hence probably in your table it doesn't have one. Create one and you should see the option. Each Single Select option would be a stack in Kanban.

**Reply by cxzxlq** [reply]
thank you！！！
